app/components/option_item/index.tsx

```diff
@@ -17,6 +18,7 @@ export type OptionItemProps = {
     icon?: string;
     info?: string;
     label: string;
+    onClose?: () => void;
```

Comment:
I would call this onRemove to be consistent and self explanatory

Reply:
Yeah, I was thinking about the name in general and if it should be even more generic than remove or close.  It's really an action icon onPress callback.  The function could do anything the caller wants.  Maybe make the name change and enhance if we decide to modify in the future?